js/packages/components/shared-components/WebViews.tsx

```diff
@@ -203,7 +211,9 @@ const ModalWebview: React.FC<{
 export const WebViews: React.FC<{ url: string }> = ({ url }) => {
 	const [isModal, setIsModal] = React.useState<boolean>(true)
 	const [isAccept, setIsAccept] = React.useState<boolean>(false)
+	const [isLoad, setIsLoad] = React.useState<boolean | undefined>()
```

Comment:
```suggestion
	const [isLoading, setIsLoading] = React.useState<boolean>()
```
- a load = une charge (dans le sens de qq chose de pesant
- `useState` adds `undefined` to the type union if used with no arguments already